ui/components/grant.js

```diff
@@ -137,16 +137,16 @@ export default function ({ id }) {
       {error && <ErrorMessage message={error} />}
       {list?.length > 0 &&
```

Comment:
this should be `list?.count`?

Reply:
or maybe on the line 24 
``const { data: { items: list } = {} } = useSWR(() => `/api/grants?resource=${destination.name}`)``

it should be 
``const { data: { items: list, count } = {} } = useSWR(() => `/api/grants?resource=${destination.name}`)``